docs/user/visual-diff.rst

```diff
@@ -12,18 +12,29 @@ and focus your review on what has changed in the current page.
 
    Example of Visual diff
 
-Enabling Visual diff
---------------------
+Using Visual diff
+-----------------
+
+Visual diff is enabled by default and it's only available on pull request builds.
+Once enabled, a new UI element appears at the top-right of the page showing a dropdown selector containing
+all the files that have changed between the base version (e.g. ``latest``) and the current pull request build.
+
+You can select any of those files from the dropdown to jump directly into that page.
+Once there, you can toggle it on/off by pressing the :guilabel:`Show diff` link from the UI element to show all the chunks and their differences highlighted with red/green background colors.
```

Comment:
```suggestion
Once there, you can toggle it on/off by pressing the :guilabel:`Show diff` link from the UI element, or pressing the `d` key if you have hotkeys enabled.
Visual diff will show all the sections that have changed and their differences highlighted with red/green background colors.
```

IIRC red/green is not great for color blind folks, but I think it's fine for now. I assume the colors are configurable in the future if needed?

Reply:
yes, we are exposing some CSS classes for this customization: https://github.com/readthedocs/addons/blob/1950a82aba2d096cdebd8b62c32db60c05b42f47/src/docdiff.document.css#L1-L16